File: supporting docs/Uncolored (Risks Assesment Tool)/src/uncolored/main/Launch.java
package uncolored.main;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;
import javafx.stage.StageStyle;

/**
 *
 * @author Benny
 */
public class Launch extends Application {

    @Override
    public void start(Stage stage) throws Exception {
        Parent root = FXMLLoader.load(getClass().getResource("/uncolored/views/Dashboard.fxml"));
        Scene scene = new Scene(root);
        stage.setScene(scene);
        stage.setTitle("Uncolored");
        stage.initStyle(StageStyle.UTILITY);
        stage.setMinWidth(980.0);
        stage.setMinHeight(680);
        stage.setWidth(980.0);
        stage.setHeight(680);
        stage.show();
    }

    public static void main(String[] args) {
        launch(args);
    }

}

File: supporting docs/UnColored .NET/src/uncolored/main/Launch.cs
namespace uncolored.main
{
	using Application = javafx.application.Application;
	using FXMLLoader = javafx.fxml.FXMLLoader;
	using Parent = javafx.scene.Parent;
	using Scene = javafx.scene.Scene;
	using Stage = javafx.stage.Stage;
	using StageStyle = javafx.stage.StageStyle;

	/// 
	/// <summary>
	/// @author Benny
	/// </summary>
	public class Launch : Application
	{

//JAVA TO C# CONVERTER WARNING: Method 'throws' clauses are not available in C#:
//ORIGINAL LINE: public void start(javafx.stage.Stage stage) throws Exception
		public override void start(Stage stage)
		{
			Parent root = FXMLLoader.load(this.GetType().getResource("/uncolored/views/Dashboard.fxml"));
			Scene scene = new Scene(root);
			stage.Scene = scene;
			stage.Title = "Uncolored";
			stage.initStyle(StageStyle.UTILITY);
			stage.MinWidth = 980.0;
			stage.MinHeight = 680;
			stage.Width = 980.0;
			stage.Height = 680;
			stage.show();
		}

		public static void Main(string[] args)
		{
			launch(args);
		}

	}

}